### PR TITLE
feat: Test class Generator

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
Supersedes #8333

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [ ] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

**Comment by lonnieezell** [comment]
I obviously did something wrong trying to push changes up after  I pulled this, but we need to make 2 changes, and then I'm good with it. 

1) Change the classNameLang string as per the suggestion
2) Add `test` => 'Test class name' to the `system/Language/en/CLI.php` file.

File: system/Commands/Generators/TestGenerator.php [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    /**
+     * Actually execute a command.
+     */
+    public function run(array $params)
+    {
+        $this->component = 'Test';
+        $this->template  = 'test.tpl.php';
+        $this->namespace = 'Tests';
+
+        $this->classNameLang = 'CLI.generator.className.entity';
```
Comment (lonnieezell) [inline system/Commands/Generators/TestGenerator.php:82]:
```suggestion
        $this->classNameLang = 'CLI.generator.className.test';
```

**Comment by lonnieezell** [comment]
@kenjis Are you going to be able to make that small change I mentioned? Or should I update my PR?

**Comment by kenjis** [comment]
@lonnieezell Please update your PR.
I think it is better that you finish your PR.

**Pull request closed without merging**